CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 4.0.0 (2019-02-08)
+
+## Improvements
+
+Addition of the reference entity end-points:
+
+- Get a single reference entity
+- Get a list of reference entities
+- Upsert a single reference entity
+- Upsert a list of reference entities
+- Get a single reference entity record
+- Get a list of reference entity records
+- Upsert a single reference entity record
+- Upsert a list of reference entity records
+- Get a single reference entity attribute
+- Get a list of reference entity attributes
+- Upsert a single reference entity attribute
+- Get a single reference entity attribute option
+- Get a list of reference entity attribute options
+- Upsert a single reference entity attribute option
+- Download a reference entity media file
+- Create a reference entity media file
+
+## BC breaks
+ 
```

Comment:
Can we say that these BC will only occur if the user extended those interfaces.

Reply:
I think it's useless, it's pretty obvious for developers. Also, we did a BC when getting an asset (for DX, issue raised by someone).